src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,114 @@
 package core.basesyntax;
 
+import java.util.Map;
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private static final int DEFAULT_INCREASE = 2;
+    private int threshold;
+    private int size;
+    private int capacity;
```

Comment:
It's redundant declaring capacity here and in constructor
To avoid this you can use code below directly in method resize() 
int capacity = capacity * DEFAULT_INCREASE;